proto/resource_manager.proto

```diff
@@ -204,7 +205,7 @@ message RunawayRule {
 
 message RunawayWatch {
   // how long would the watch last
-  uint64 lasting_duration_ms = 1;
+  int64 lasting_duration_ms = 1;
```

Comment:
why not use 0 as unlimited

Reply:
In TiDB, ResourceGroupRunawaySettings is defined as follow,
```
type ResourceGroupRunawaySettings struct {
	ExecElapsedTimeMs uint64            `json:"exec_elapsed_time_ms"`
	Action            RunawayActionType `json:"action"`
	WatchType         RunawayWatchType  `json:"watch_type"`
	WatchDurationMs   int64             `json:"watch_duration_ms"`
}
```
I should use WatchDurationMs to check whether `Watch` is defined.
But I can use a pointer for `Watch`.   Which do you think is better

Reply:
maybe add a none watch type?

Reply:
+1